packages/browser-lib/README.md

```diff
@@ -0,0 +1,28 @@
+# `@sa11y/browser-lib`
+
+Provides a minified version of selected `@sa11y` libraries to be injected into a browser (using webdriver) and executed from integration testing workflows. This allows for reuse of the `@sa11y` libraries across unit and integration testing workflows.
+
+Code in this package should be limited only to wrappers required to facilitate execution in browser environment. All primary code should be added to `@sa11y` libraries.
```

Comment:
I'm a little unclear when you would use this package vs. the wdio package. I see the example usage below looks like it's even `wdio` sample code.

Reply:
It is to be used in testing workflows without webdriverIO e.g. Java. Added clarification to doc. Please let me know if that answers your question.

Reply:
Ok got it. That makes sense, but I still find it a little confusing that the sample below is in Javascript if the main target is Java. It'd be nice to see a more complete example authored in Java, even if the test cases in this project are still Javascript. 

If I was a consumer trying to integrate this into my Java test suite, it would take some exploration still to figure out how exactly to use it.

Reply:
Agree, was thinking the same. Added Java example.